File: Routes/speed.py
from flask_restplus import Namespace, Resource
from Conversions import speed

ns_speed = Namespace('Speed', description='Speed calculations')


@ns_speed.route('/MPH/<string:input>/ToKnots')
class Conversions(Resource):
    def get(self, input):
        try:
            input = float(input)
            return speed.mph_to_knots(input)
        except ValueError:
            return "Can't convert {} to number".format(input), 400


@ns_speed.route('/Knots/<string:input>/ToMPH')
class Conversions(Resource):
    def get(self, input):
        try:
            input = float(input)
            return speed.knots_to_mph(input)
        except ValueError:
            return "Can't convert {} to number".format(input), 400


@ns_speed.route('/Knots/<string:input>/ToMach')
class Conversions(Resource):
    def get(self, input):
        try:
            input = float(input)
            return speed.knots_to_mach(input)
        except ValueError:
            return "Can't convert {} to number".format(input), 400


@ns_speed.route('/Mph/<string:input>/ToMach')
class Conversions(Resource):
    def get(self, input):
        try:
            input = float(input)
            return speed.mph_to_mach(input)
        except ValueError:
            return "Can't convert {} to number".format(input), 400

File: routes/speed.js
const apiBasePath = '/api/v2/Speed/';

function mphToKnots(input) {
  return input * 0.8689758;
}

function knotsToMph(input) {
  return input * 1.15078;
}

function knotsToMach(input) {
  return input * 0.0015117824339371;
}

function mphToMach(input) {
  return input * 0.001303;
}

export const speedRoutes = [
  {
    method: 'GET',
    path: `${apiBasePath}mphToKnots/{input}`,
    handler: (request) => mphToKnots(request.params.input)
  },
  {
    method: 'GET',
    path: `${apiBasePath}knotsToMph/{input}`,
    handler: (request) => knotsToMph(request.params.input)
  },
  {
    method: 'GET',
    path: `${apiBasePath}knotsToMach/{input}`,
    handler: (request) => knotsToMach(request.params.input)
  },
  {
    method: 'GET',
    path: `${apiBasePath}mphToMach/{input}`,
    handler: (request) => mphToMach(request.params.input)
  }
];
